### PR TITLE
Rename and rebranding #9

### PR DESCRIPTION
**Pull request by mkubala**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,56 +13,38 @@ lazy val core = project
...
     previewPath := (Paradox / siteSubdirName).value,
-    Preprocess / siteSubdirName := s"api/akka-persistence-jdbc/${if (isSnapshot.value) "snapshot"
+    Preprocess / siteSubdirName := s"api/akka-persistence-postgres/${if (isSnapshot.value) "snapshot"
       else version.value}",
     Preprocess / sourceDirectory := (LocalRootProject / ScalaUnidoc / unidoc / target).value,
-    Paradox / siteSubdirName := s"docs/akka-persistence-jdbc/${if (isSnapshot.value) "snapshot" else version.value}",
+    Paradox / siteSubdirName := s"docs/akka-persistence-postgres/${if (isSnapshot.value) "snapshot" else version.value}",
     Compile / paradoxProperties ++= Map(
         "project.url" -> "https://doc.akka.io/docs/akka-persistence-jdbc/current/",
         "canonical.base_url" -> "https://doc.akka.io/docs/akka-persistence-jdbc/current",
```
Comment (devcateu) [inline build.sbt:44]: jdbc

File: core/src/main/resources/reference.conf [hunk 2]
```diff
@@ -104,7 +104,7 @@ akka-persistence-jdbc {
 }
 
 # the akka-persistence-journal in use
-pg-journal {
+postgres-journal {
   class = "akka.persistence.postgres.journal.JdbcAsyncWriteJournal"
```
Comment (devcateu) [inline core/src/main/resources/reference.conf:108]: I am not sure, but shouldnt we change that value to PostgresAsyncWriteJournal

File: core/src/main/resources/reference.conf [hunk 3]
```diff
@@ -344,13 +344,13 @@ pg-snapshot-store {
 }
 
 # the akka-persistence-query provider in use
-pg-read-journal {
+postgres-read-journal {
   class = "akka.persistence.postgres.query.JdbcReadJournalProvider"
```
Comment (devcateu) [inline core/src/main/resources/reference.conf:348]: JDBC -> Postgres?

File: core/src/main/resources/reference.conf [hunk 4]
```diff
@@ -244,7 +244,7 @@ pg-journal {
 }
 
 # the akka-persistence-snapshot-store in use
-pg-snapshot-store {
+postgres-snapshot-store {
   class = "akka.persistence.postgres.snapshot.JdbcSnapshotStore"
```
Comment (devcateu) [inline core/src/main/resources/reference.conf:248]: JDBC -> Postgres?

File: build.sbt (hunk 1 above)
Comment (mkubala) [reply build.sbt:44]: This is a url to the Akka documentation page. We will eventually replace it with our github page. Also I'm going to get rid off Paradox.

File: core/src/main/resources/reference.conf (hunk 4 above)
Comment (mkubala) [reply core/src/main/resources/reference.conf]: Done

File: core/src/main/resources/reference.conf (hunk 2 above)
Comment (mkubala) [reply core/src/main/resources/reference.conf]: Agree, done!

**Pull request merged**